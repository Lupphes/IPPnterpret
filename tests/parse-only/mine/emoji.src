.IPPcode20
WRITE string@c😍unter