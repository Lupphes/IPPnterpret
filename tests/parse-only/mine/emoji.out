<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode20">
 <instruction order="1" opcode="WRITE">
  <arg1 type="string">c😍unter</arg1>
 </instruction>
</program>